apps/look-and-feel-stories/src/Accordion/Accordion.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Accordion,
+  accordionVariants,
+  AccordionVariants,
+} from "@axa-fr/design-system-apollo-react/lf";
+import { Meta, StoryObj } from "@storybook/react";
+import type { ComponentProps } from "react";
+import bank from "@material-symbols/svg-700/rounded/account_balance-fill.svg";
+
+const meta: Meta<typeof Accordion> = {
+  component: Accordion,
+  title: "Components/Accordion (LF)",
```

Comment:
Pas besoin de préciser, c'est dans un storybook dédié à l&f
```suggestion
  title: "Components/Accordion",
```

Reply:
Juste pour mettre en évidence la différence entre l'ancien et le nouveau composant, puisque j'ai gardé les deux stories.